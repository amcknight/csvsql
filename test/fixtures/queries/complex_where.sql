SELECT first_name, last_name
FROM test/fixtures/tables/people
WHERE age < 15 * 2 AND last_name = "Mc" ++ "Knight"
