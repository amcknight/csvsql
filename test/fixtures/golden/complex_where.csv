first_name, last_name
July, McKnight
